catalog/spatial/geocoding/spatial-geocoding-geocoder/src/main/java/org/codice/ddf/spatial/geocoder/GazetteerGeoCoder.java

```diff
@@ -49,21 +45,10 @@ public GeoResult getLocation(final String location) {
             final List<GeoEntry> topResults = geoEntryQueryable.query(location, 1);
 
             if (topResults.size() > 0) {
-                final GeoEntry topResult = topResults.get(0);
-                final String name = topResult.getName();
-                final double latitude = topResult.getLatitude();
-                final double longitude = topResult.getLongitude();
-                final String featureCode = topResult.getFeatureCode();
-                final long population = topResult.getPopulation();
-
-                return GeoResultCreator.createGeoResult(name,
-                        latitude,
-                        longitude,
-                        featureCode,
-                        population);
+                return GeoResultCreator.createGeoResult(topResults.get(0));
             }
         } catch (GeoEntryQueryException e) {
-            LOGGER.debug("Error querying the local GeoNames index", e);
+            LOGGER.debug("Error querying GeoNames", e);
         }
 
         return null;
```

Comment:
Added. Is returning an `Optional<>` always just better, more idiomatic Java nowadays?